Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -236,6 +235,46 @@ Describe "Remove-GcbqDataset" {
     }
 }
 
-#TODO(ahandley): Describe "Set-GcbqDataset".
+Describe "Set-GcbqDataset" {
+
+    It "should update trivial metadata fields via pipeline (not DatabaseId)" {
+        try {
+            New-GcbqDataset "test_dataset_id1" -Name "Testdata" -Description "Some interesting data!" -Expiration $oneDaySec
+            $data = Get-GcbqDataset "test_dataset_id1"
+            $data.FriendlyName = "Some Test Data"
+            $data.Description = "A new description!"
+            $data.DefaultTableExpirationMs = $threeDayMs
+            $data | Set-GcbqDataset -outvariable data
```

Comment:
You can also do `$data = $data | Set-GcbqDataset`. If you want to keep it as is, I would suggest capitalizing `-OutVariable`.